src/rust/mangaworld/sources/mangaworld/res/source.json

```
@@ -4,7 +4,7 @@
 		"lang": "it",
 		"name": "MangaWorld",
 		"version": 2,
```

source version needs to be bumped, check the checklist
```suggestion
		"version": 3,
```